static/themes/common/base.css

```diff
@@ -315,12 +315,13 @@
     border-bottom: 1px solid var(--brand-midtone);
 }
 
-.kiwi-userbox .kiwi-userbox-actions .kiwi-userbox-action {
+.kiwi-userbox-action {
     border: 1px solid var(--brand-default-fg);
     color: var(--brand-default-fg);
 }
 
-.kiwi-userbox .kiwi-userbox-actions .kiwi-userbox-action:hover {
+.kiwi-userbox-action:hover,
+.kiwi-userbox-plugin-actions a:hover {
```

Comment:
Should this be `.kiwi-userbox-plugin-action:hover`?

Reply:
This is for the same reason as above :)